src/Tests/Microsoft.NET.Sdk.Razor.Tests/StaticWebAssets/MergeConfigurationPropertiesTest.cs

```diff
@@ -55,6 +55,34 @@ public void MergesProjectConfigurationWithProjectReferenceWhenMatchingReferenceF
             config.GetMetadata("AdditionalPublishPropertiesToRemove").Should().Be("TargetFramework;RuntimeIdentifier");
         }
 
+        [Fact]
+        public void MergesProjectConfigurationWithProjectReference_UsesOSCasingForMathing()
```

Comment:
Typo `MergesProjectConfigurationWithProjectReference_UsesOSCasingForMatching` instead of `MergesProjectConfigurationWithProjectReference_UsesOSCasingForMathing`

Reply:
```suggestion
        public void MergesProjectConfigurationWithProjectReference_UsesOSCasingForMatching()
```